lib/gui/etcher.js

```diff
@@ -41,12 +41,13 @@ electron.app.on('ready', () => {
   // Prevent flash of white when starting the application
   // https://github.com/atom/electron/issues/2172
   mainWindow.webContents.on('did-finish-load', () => {
+    const FLASH_TIMEOUT_MS = 100;
```

Comment:
Makes sense. I've changed it to `WEBVIEW_LOAD_TIMEOUT_MS`, since the timeout is necessary because the Chromium WebView takes a bit to load completely.